lib/hydra/pcdm/services/collection/get_related_objects.rb

```diff
@@ -10,8 +10,7 @@ class GetRelatedObjectsFromCollection
 
     def self.call( parent_collection )
       raise ArgumentError, "parent_collection must be a pcdm collection" unless Hydra::PCDM.collection? parent_collection
-
-      parent_collection.related_objects
+      parent_collection.related_objects.to_a
```

Comment:
Two reasons for the related_object.to_a
1. related_objects.include? raises an error in some cases  (This should be fixed when issues https://github.com/projecthydra-labs/activefedora-aggregation/issues/36 is fixed)
2. col1.collections and col1.objects in collection_behaviors each returns an Array and is surfaced through services as GetCollectionsFromCollection and GetObjectsFromCollection.  For consistency in processing, col1.related_objects surfaced through GetRelatedObjectsFromCollection should return an Array.  

NOTE: Same applies for related objects in pcdm objects.